input/system/azure/system.go

```diff
@@ -286,6 +286,7 @@ func getFirstMetricValue(metric *azquery.Metric) (metricValue *azquery.MetricVal
 	for _, timeSeriesElement := range metric.TimeSeries {
 		for _, mValue := range timeSeriesElement.Data {
 			metricValue = mValue
+			return
 		}
 	}
 	return
```

Comment:
Why not
```suggestion
        if len(metric.TimeSeries) == 0 || len(metric.TimeSeries[0].timeSeriesElement.Data) == 0 {
                return
        }
        return metric.TimeSeries[0].timeSeriesElement.Data[0]
```
?

Reply:
I see, yeah I agree that it can be more straight forward. Just for the context, this loop was originally coming from "what we should be getting values" like this example:
https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/monitor/azquery/example_test.go#L261

Though, above example is to print out all data, hence the loop. Since we narrow down the usage here and only returning the first one, we can simplify this.